overrides/remote_gahp

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+################################################################################
+# blahp-over-ssh shim script
+#
+# command [options] [user@]hostname [options] command [remote arguments]
+#
+# Options:
+#   see usage string 
+#
+# Arguments:
+#   mandatory: hostname, command
+#
+# Remote arguments are passed along
+#
+# Assumes public/private key pair is already created and exists on host
+#
+# Exits with 0 on success, 1 if wrong arguments, < 0 if ssh-add failed, > 0 if ssh failed
+################################################################################
+
+#Defaults
+REMOTE_CMD=""
+REMOTE_HOSTNAME=""
+REMOTE_USER=`whoami`
+# Fix the home directory when spawned from a root process
+unset HOME
+export HOME=`echo ~`
```

Comment:
Does this work??

Reply:
Presumably yes because this is what's in upstream :shrug: